### PR TITLE
Nginx template improvements

### PR DESCRIPTION
**Pull request by MarekBenjamin**
- Updated sample-nginx.conf to use php7.4-fpm (was 7.0)
- Removed double hard coded friendica.example.net declatation by replacing defining it with already defined `$server_name` variable
- Added `sample-nginx-cerbot.config` to be used directly with `cerbot --nginx`

### CODE REVIEW

**Review by MrPetovan — COMMENTED** [review]
Is the `mods/sample-nginx-certbot.config` file intended to be empty?

**Comment by MarekBenjamin** [comment]
Not at all! Gonna check that shortly

24.11.2022 20:45:42 Hypolite Petovan ***@***.***>:

**Comment by MarekBenjamin** [comment]
Updated with the actual content,

File: mods/sample-nginx-certbot.config [hunk 1]
```diff
@@ -0,0 +1,128 @@
+##
+# Friendica Nginx configuration template to be autocnfgiured with cerbot
+# nased on sample-nginx.config by Olaf Conradi
```
Comment (nupplaphil) [inline mods/sample-nginx-certbot.config]:
```suggestion
# Friendica Nginx configuration template to be autoconfigured with certbot
# based on sample-nginx.config by Olaf Conradi
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]

**Comment by nupplaphil** [comment]
Thank you for your work :-)

**Pull request merged**